index.html

```diff
@@ -92,16 +92,20 @@
     <section>
       <h2>Introduction</h2>
       <p>
-        There is a need in the media industry for an API to support metadata
-        events synchronized to audio or video media, specifically for both
-        <a>out-of-band</a> event streams and <a>in-band</a> discrete events
-        (for example, MPD and <code>emsg</code> events in MPEG-DASH).
-        These <a>media timed events</a> can be used to support use cases
-        such as dynamic content replacement, ad insertion, or presentation of
-        supplemental content alongside the audio or video, or more generally,
-        making changes to a web page, or executing application code triggered
-        from JavaScript events, at specific points on the <a>media timeline</a>
-        of an audio or video media stream.
+        There is a need in the media industry for an API to support <a>timed
+        metadata</a>, i.e., metadata that is synchronized to audio or video
```

Comment:
Again I would prefer "data" to "metadata" here, for precision. I would consider metadata to be a subset of data, in their general definitions.

Reply:
I have rewritten the Introduction to be more explicit w.r.t timed metadata.